packages/office-addin-debugging/src/start.ts

```diff
@@ -52,9 +61,32 @@ export async function isPackagerRunning(statusUrl: string): Promise<boolean> {
 export function parseAppType(text: string): AppType | undefined {
     switch (text) {
         case "desktop":
+        case "macos":
+        case "win32":
             return AppType.Desktop;
         case "web":
             return AppType.Web;
+        case "ios":
+        case "android":
+            throw new Error(`Platform type ${text} not currently supported for debugging`);
+        default:
+            return undefined;
+    }
+}
+
+export function parseAppPlatform(text: string): AppPlatform | undefined {
+    switch (text) {
+        case "desktop":
+            return process.platform === "win32" ? AppPlatform.Win32 : AppPlatform.MacOS;
+        case "macos":
+            return AppPlatform.MacOS;
+        case "web":
+            return AppPlatform.Web;
+        case "win32":
+            return AppPlatform.Win32;
+        case "ios":
+        case "android":
+            throw new Error(`Platform type ${text} not currently supported for debugging`);
```

Comment:
As I said before, implement this for ios and android. Move the error check elsewhere.